internal/linker/link_node_modules.bzl

```diff
@@ -59,9 +65,11 @@ def write_node_modules_manifest(ctx, extra_data = []):
 
     mappings = {
         # We always map the workspace to itself to support absolute require like
-        # import from 'my_wksp/path/to/file'
-        # and it's always a bin dependency, TODO: let user control via package.json
-        ctx.workspace_name: ["bin", ctx.workspace_name],
+        # import from 'my_wksp/path/to/file' and it's always links to the bin_dir.
+        # If source files need to be required then they can be copied to the bin_dir
+        # with copy_to_bin.
+        # TODO: let user control the link path via package.json.
```

Comment:
what would this look like? seems like the path is always the directory containing package.json?

Reply:
I don't know. That TODO was there from the original linker impl. Perhaps it is referring to the the root package.json to control how to root workspace is linked. Seems out of date at this point so I'll remove the TODO.